src/main/java/com/teragrep/pth_06/planner/walker/ConditionWalker.java

```diff
@@ -68,27 +73,37 @@ public class ConditionWalker extends XmlWalker<Condition> {
     // Default query is full
     private boolean streamQuery = false;
     private final DSLContext ctx;
```

Comment:
is it possible to make these final?

Reply:
made final also ValidElement class is final now

Reply:
what about `streamQuery` and `bloomTermId`?

Reply:
making `streamQuery` final would be possible using separate walkers for stream and journal conditions but that would need some refactoring of testing that I would do in another PR.

Reply:
`bloomTermId` would also a bigger refactor to make work with walker traverse on a different PR

Reply:
will make issues for  both